pkg/cmd/roachtest/test_runner.go

```diff
@@ -664,13 +712,6 @@ func (r *testRunner) runTest(
 			if teamCity {
 				shout(ctx, l, stdout, "##teamcity[testFailed name='%s' details='%s' flowId='%s']",
 					t.Name(), teamCityEscape(output), runID)
-
-				// Copy a snapshot of the testrunner's log to the test's artifacts dir
-				// so that we collect it below.
-				cp := exec.Command("cp", testRunnerLogPath, t.ArtifactsDir())
```

Comment:
@andreimatei fyi, after this change, there will be only one copy of `test_runner` log in the top-level artifacts dir.